### ISSUE TITLE
Error with logger on exptract_promoter

### ISSUE THREAD

**Issue by homonecloco** [EI-CoreBioinformatics/mikado#404]
I am trying to extract the promoter coordinates, I am first testing the tool for this function, to see if the output produces the coordinates.  However, I get this issue:

```
% extract_promoter_regions.py --exclude-utr --gzip --no-neighbours -d2000 -o promoters_2k_refseq1.1 -l logextract.log -lv INFO  Triticum_aestivum.IWGSC.dna_sm.toplevel.fa  gene_list.txt

Traceback (most recent call last):
  File "/usr/local/bin/extract_promoter_regions.py", line 176, in <module>
    main()
  File "/usr/local/bin/extract_promoter_regions.py", line 36, in main
    logger = Mikado.utilities.log_utils.create_logger_from_conf({"log_settings": {"log": args.log,
  File "/usr/local/lib/python3.9/site-packages/Mikado/utilities/log_utils.py", line 146, in create_logger_from_conf
    log_settings = conf.log_settings
AttributeError: 'dict' object has no attribute 'log_settings'
```

My guess is that there was a change in how the logger receives the configuration since the time when this tool was written

**Comment by ljyanesm** [comment]
Dear @homonecloco,

It seems like this script needs a bit of updating, I'll have a look and get back to you.

**Comment by ljyanesm** [comment]
Fixed in 2.3.0
